website/content/en/highlights/2021-11-16-implicit-namespacing.md

```diff
@@ -0,0 +1,70 @@
+---
+date: "2021-11-16"
+title: "Automatic namespacing for config files"
+description: "A guide that addresses the new automatic namespacing functionality"
```

Comment:
The description should just directly say what the highlight is about. No need for "a guide that..." or a preamble of that sort.